server/src/main/java/org/elasticsearch/action/search/TransportSearchAction.java

```diff
@@ -1317,30 +1317,32 @@ public SearchPhase newSearchPhase(
                             false,
                             threadPool,
                             clusters
-                        );
-                        action.start();
-                    })
-                );
-            } else {
-                // for synchronous CCS minimize_roundtrips=false, use the CCSSingleCoordinatorSearchProgressListener
-                // (AsyncSearchTask will not return SearchProgressListener.NOOP, since it uses its own progress listener
-                // which delegates to CCSSingleCoordinatorSearchProgressListener when minimizing roundtrips)
-                if (clusters.isCcsMinimizeRoundtrips() == false
-                    && clusters.hasRemoteClusters()
-                    && task.getProgressListener() == SearchProgressListener.NOOP) {
-                    task.setProgressListener(new CCSSingleCoordinatorSearchProgressListener());
-                }
-                final SearchPhaseResults<SearchPhaseResult> queryResultConsumer = searchPhaseController.newSearchPhaseResults(
-                    executor,
-                    circuitBreaker,
-                    task::isCancelled,
-                    task.getProgressListener(),
-                    searchRequest,
-                    shardIterators.size(),
-                    exc -> searchTransportService.cancelSearchTask(task, "failed to merge result [" + exc.getMessage() + "]")
+                        ).start()
+                    )
                 );
+            }
```

Comment:
LOL, had to turn of "whitespace" in github to see that this was just a nice clean up as we were returning out of the `if` anyways!